--- a/caching/Cache.php
+++ b/caching/Cache.php
@@ -194,7 +194,7 @@
         $results = [];
         foreach ($keyMap as $key => $newKey) {
             $results[$key] = false;
-            if (isset($values[$newKey])) {
+            if (isset($values[$newKey]) && $values[$newKey] !== false) {
                 if ($this->serializer === false) {
                     $results[$key] = $values[$newKey];
                 } else {
