diff --git a/node_modules/eslint/lib/rules/no-unused-vars.js b/node_modules/eslint/lib/rules/no-unused-vars.js
index 8094de5..48df3aa 100644
--- a/node_modules/eslint/lib/rules/no-unused-vars.js
+++ b/node_modules/eslint/lib/rules/no-unused-vars.js
@@ -507,7 +507,7 @@ module.exports = {
             const childScopes = scope.childScopes;
             let i, l;
 
-            if (scope.type !== "global" || config.vars === "all") {
+            if (scope.type !== "TDZ" && (scope.type !== "global" || config.vars === "all")) {
                 for (i = 0, l = variables.length; i < l; ++i) {
                     const variable = variables[i];
 
